src/components/Common/Module/FieldPicker.vue

```diff
@@ -1,49 +1,143 @@
 <template>
-    <fieldset class="form-group">
-      <div class="fields d-flex">
-        <div class="available">
-          <label>{{ $t('selector.available') }}</label>
-          <b-button @click.prevent="selectedFields = [...allFields]" variant="link" class="float-right">{{ $t('selector.selectAll') }}</b-button>
-          <draggable
-            class="drag-area border"
-            :list.sync="availableFields"
-            :group="group"
+    <fieldset
+      :disabled="disabled"
+      class="form-group"
+    >
+      <b-form-group
+        :label="$t('block:recordList.fields')"
+        class="mt-3 h5"
+        label-cols="2"
+      >
+        <b-input-group
+          class="w-25"
+        >
+          <b-input
+            v-model.trim="query"
+            type="search"
+            :placeholder="$t('module:searchPlaceholder')"
+            class="border-right-0 text-truncate"
+          />
+          <b-input-group-append>
+            <b-input-group-text class="text-primary bg-white border-left-0">
+              <font-awesome-icon
+                :icon="['fas', 'search']"
+              />
+            </b-input-group-text>
+          </b-input-group-append>
+        </b-input-group>
+      </b-form-group>
+      <b-container
+        class="d-flex px-0"
+      >
+        <b-row
+          class="w-50"
```

Comment:
Remove width since it defeats the whole purpose of rows and columns
Not sure you understood me before but, both of the picker should be in one row, and each picker in its own column